timely/src/progress/frontier.rs

```diff
@@ -229,6 +229,16 @@ impl<T: Clone> Clone for Antichain<T> {
 
 impl<T: TotalOrder> TotalOrder for Antichain<T> { }
 
+impl<T: Ord+std::hash::Hash> std::hash::Hash for Antichain<T> {
+    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
+        let mut temp = self.elements.iter().collect::<Vec<_>>();
```

Comment:
this temp Vec is unfortunate, should I be doing something different? maybe impl Hash for MutableAntichain (which requires that T be Ord) instead?

Reply:
This seems fine to me, and is in-line with how we do equality. It just means that `Hash` is not super cheap, but that isn't the end of the world imo. A different take is that we should maintain antichains in normal form (sorted, say), which is a different amount of disruptive.

Reply:
Okay then it works for me, too! It's still much faster than the alternative (repeatedly iterating through a list to call Eq). Just wanted to make sure I wasn't missing something.